ocp_resources/node_network_configuration_policy.py

```diff
@@ -385,7 +385,7 @@ def wait_for_status_success(self):
         # if we get here too fast there are no conditions, we need to wait.
         self.wait_for_configuration_conditions_unknown_or_progressing()
 
-        samples = TimeoutSampler(wait_timeout=480, sleep=1, func=self.status)
+        samples = TimeoutSampler(wait_timeout=540, sleep=1, func=self.status)
```

Comment:
please get wait_timeout arg with default 480 and pass it when needed.

Reply:
ACK